src/hexkit/providers/s3/testutils/_fixtures.py

```diff
@@ -111,19 +110,66 @@ async def populate_file_objects(self, file_objects: list[FileObject]):
             self.storage, bucket_fixtures=[], object_fixtures=file_objects
         )
 
-    async def empty_buckets(self, buckets_to_exclude: Optional[list[str]] = None):
-        """Clean the test artifacts or files from the populated buckets."""
-        for bucket in self.get_buckets().difference(buckets_to_exclude or []):
+    async def empty_buckets(
+        self,
+        *,
+        buckets: Optional[Union[str, list[str]]] = None,
+        exclude_buckets: Optional[list[str]] = None,
```

Comment:
```suggestion
        exclude_buckets: Optional[Union[str, list[str]]] = None,
```
Either this type should be updated to include `str`, or the `isinstance(exclude_buckets, str)` check should be deleted below

Reply:
Improve type hints